doc/source/install/index.rst

```diff
@@ -2,34 +2,330 @@
 Installation
 ============
 
-Each OS version has it's own package names, versions, bugs. However there is a repository,
-which tries to address those issues (at least for RPM based distributions) - https://build.opensuse.org/project/show/Cloud:OTC:Tools:OpenStack.
-RPMs for most often used OSs are prepared and can be taken there. 
+There are several ways to install python-otcextensions to enhance the native ``openstack`` CLI client and to extend the OpenStack SDK to cover the additional Open Telekom Cloud services providing a larger functionality on top of OpenStack.
 
-With a pip package can be installed the following way (please remember, it is not the latest development state). At the command line::
+The easiest way is to use the Python pip installer which is working distribution independent and can be used in an isolated virtual environment.
+There are also ready-made installation packages for various operating systems which have their own versions, package names and sometimes bugs. A repository based on openSUSE's build services tries to cover these issues which is available under: https://build.opensuse.org/project/show/Cloud:OTC:Tools:OpenStack.
 
-    $ sudo yum install python-pip
-    $ sudo pip install otcextensions --upgrade
+Package Overview
+----------------
```

Comment:
Overview of which package is it now?

Reply:
Openstack sdk, otcextensions and openstackclient (the main packages are meant.

Reply:
while we are describing installation of otcextensions (install instructions of the package). It might be just confusing

Reply:
Changed header to clearify the focus.